pkg/secrethub/secret_version.go

```diff
@@ -21,6 +21,7 @@ var (
 	ErrSecretTooBig         = errClient.Code("secret_too_big").Error(fmt.Sprintf("maximum size of a secret is %s", units.BytesSize(MaxSecretSize)))
 	ErrEmptySecret          = errClient.Code("empty_secret").Error("secret is empty")
 	ErrCannotWriteToVersion = errClient.Code("cannot_write_version").Error("cannot (over)write a specific secret version, they are append only")
+	ErrSecretNotFound       = errClient.Code("secret_not_found").ErrorPref("cannot find secret: %s, error: %v")
```

Comment:
If it's an `ErrorPref`, should it be public?

Reply:
I think it should, just to be consistent with the other errors in the client.